### PR TITLE
Fix the hash filtering

### PR DESCRIPTION
**Pull request by tshedor**
Update hash-parsing library to Fryr and fix cookie/hash filtering on load. Hotfix.

### CODE REVIEW

**Comment by jamesmk** [comment]
@tshedor can you check on the failing tests?

**Comment by cpitkin** [comment]
This is a test comment

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - enhancements
 - bugs
+    + Update hash-parsing library to Fryr and fix cookie/hash filtering on load
```
Comment (cpitkin) [inline CHANGELOG.md]: Testing inline comment

Comment (cpitkin) [inline CHANGELOG.md]: inline

Comment (cpitkin) [inline CHANGELOG.md]: code comment test

**Comment by cpitkin** [comment]
@jamesmk i think it is working agin

**Comment by tshedor** [comment]
@jamesmk tests are passing. It's been confirmed.

**Pull request merged**